### PR TITLE
RadioShack

### PR DESCRIPTION
**Pull request by radioshackcreator**
Adding RadioShack, a Uniswap v2 fork, with a 'starfish' swap pair topology. Learn more here:
https://docs.radioshack.org

App is live here on Polygon and BSC (Avalanche and Ethereum coming soon):
https://app.radioshack.org/swap

### CODE REVIEW

**Review by ColonelJ — CHANGES_REQUESTED** [review]
Hi, if your DEX doesn't need any special logic to deal with dynamic fees or something else, then you don't need to add any `radioshack.ts` you can simply add your forks to the `config.ts`.  You are using a `'swapFee'` function in `radioshack.ts` which doesn't seem to exist in your ABI, so I don't think you have dynamic fees.  So I think nearly all your changes are not needed (don't need your ABI, dedicated config etc.).  Just add to the main `config.ts` your forks and keep the tests you've already done.

File: src/dex/uniswap-v2/uniswap-v2-e2e-bsc.test.ts [hunk 1]
```diff
@@ -633,6 +633,152 @@ describe('UniswapV2 E2E BSC', () => {
     });
   });
 
+  describe('RadioShack', () => {
+    const dexKey = 'RadioShack';
+
+    describe('Simpleswap', () => {
+      it('BNB -> TOKEN', async () => {
+        await testE2E(
+          tokens.WBNB,
```
Comment (shresthagrawal) [inline src/dex/uniswap-v2/uniswap-v2-e2e-bsc.test.ts:642]:
```suggestion
          tokens.BNB,
```

File: src/dex/uniswap-v2/uniswap-v2-e2e-bsc.test.ts [hunk 2]
```diff
@@ -633,6 +633,152 @@ describe('UniswapV2 E2E BSC', () => {
...
+          tokens.USDC,
+          holders.WBNB,
```
Comment (shresthagrawal) [inline src/dex/uniswap-v2/uniswap-v2-e2e-bsc.test.ts:644]:
```suggestion
          holders.BNB,
```

**Comment by shresthagrawal** [comment]
The e2e tests are failing because the pools doesn't seem to exist for the pairs! Would be good if you can fix the tests and check that it works. You can do that by having the appropriate env variables and running `npx jest -t 'radioshack' src/dex/uniswap-v2/uniswap-v2-e2e-bsc.test.ts` for bsc and `npx jest -t 'radioshack' src/dex/uniswap-v2/uniswap-v2-e2e-polygon.test.ts`for mainnet

**Review by shresthagrawal — APPROVED** [review]
LGTM

**Review by ColonelJ — APPROVED** [review]

**Pull request merged**